package.json

```diff
@@ -28,7 +28,8 @@
         "electron-updater": "^6.3.4"
     },
     "optionalDependencies": {
-        "@vencord/venmic": "^6.1.0"
+        "@vencord/venmic": "^6.1.0",
+        "@homebridge/dbus-native": "0.6.0"
```

Comment:
adding a new library that adds more than 200kb of javascript just for one single dbus call is pretty bad